src/routes/events.js

```diff
@@ -45,12 +45,19 @@ router.put(
     autherizationMiddleware(EventModel),
     eventsController.updateEvent
 );
-
+// ADD USER TO EVENT
 router.post(
     '/:id/volunteers',
```

Comment:
Instead of naming this endpoint `volunteers` and the other one `volunteer`, I suggest that you keep this one as `volunteer` and name the other `un-volunteer`. 

In fact, there is another way to do this. You would have just one endpoint called to volunteer and you would require the user to pass a boolean body parameter (you can call it `operationType`) to specify the action (volunteer/unvolunteer). Then, inside your controller, you check the operation type and perform the required actions.